src/App.vue

```diff
@@ -59,6 +65,11 @@
       return currencies[this.snippetConfig.currency!];
     }
 
+    @XProvide('queriesPreviewInfo')
```

Comment:
Should we watch this property? What would happen if `queriesPreview` is not defined initially in the `snippetConfig` and then is added programmatically through the `InterfaceX.setSnippetConfig` callback?

Reply:
That shouldn't be needed because `setSnippetConfig` adds the properties reactively and the `XProvide` decorator keeps the queries previews also reactive.
I have just tested it, and it is working fine.